### DISCUSSION TITLE
Need guidance in learning XML

### DISCUSSION THREAD

**Discussion by wisteria86** [community/community#47628] [Other Feature Feedback, Questions, & Ideas]
Hello I am AbdulRehman. It's nice to meet you all.
The reason I made this post is because I need help in learning XML.
Do you know any free resources from where I can learn ? Like online pdf books , video lectures or online webpages containing information on XML.

The reason why I want to learn XML is because I want to develop my application interface.
I am a java application developer.
Thank you for your time!

**Comment by SilasRodrigues19** [comment]
Hey @AbdulRehman775 

This link below has a nice content about `Java XML`:

[Learn Java XML](https://www.tutorialspoint.com/java_xml/index.htm)

**Comment by marianne-s** [comment]
Hi @AbdulRehman775 , an other nice content on XML in a step by step description:

https://learnxinyminutes.com/docs/xml/
